examples/deno-oak/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM denoland/deno:1.15.3
```

Comment:
```suggestion
FROM denoland/deno:latest
```

Reply:
Well it's better to provide an explicit version in Dockerfiles to prevent updates breaking the image.

Reply:
Updates won't break this, as Deno seems to promise backwards compatibility. And as always, performance updates come, and, who doesn't like good performance.

PS. What would break native TS / JS syntax?